backup_and_restore/application_backup_and_restore/installing/installing-oadp-ibm-cloud.adoc

```diff
@@ -0,0 +1,34 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-oadp-ibm-cloud"]
+= Configuring the OpenShift API for Data Protection with {ibm-cloud-title}
```

Comment:
```suggestion
= Configuring the {oadp-full} with {ibm-cloud-title}
```

Note: This attribute might not render in the preview. In that case, shift `include::_attributes/common-attributes.adoc[]` above the assembly heading.